### ISSUE TITLE
[Remote rendering problem] Stray Android ARM64 meta files cause Unity warnings on every script recompilation

### ISSUE THREAD

**Issue by NoTuxNoBux** [Azure/azure-remote-rendering#74]
**Describe the problem**

Several meta files are packaged in the build that don't seem to match any actual files, in particular the ARM64 Android DLLs appear to be missing, but the meta files and folder are still present. This results in Unity issuing warnings on every script recompilation, cluttering the console with spurious messages.

**Provide additional information**

The messages Unity prints are:

```
A meta data file (.meta) exists but its asset 'Packages/com.microsoft.azure.remote-rendering/Plugins/Android/ARM64/libRemoteRenderingClient.so' can't be found. When moving or deleting files outside of Unity, please ensure that the corresponding .meta file is moved or deleted along with it.

Couldn't delete Packages/com.microsoft.azure.remote-rendering/Plugins/Android/ARM64/libRemoteRenderingClient.so.meta because it's in an immutable folder.

A meta data file (.meta) exists but its asset 'Packages/com.microsoft.azure.remote-rendering/Plugins/Android/ARM64/libXrApiLayer_MicrosoftHolographicHybridRemoting.so' can't be found. When moving or deleting files outside of Unity, please ensure that the corresponding .meta file is moved or deleted along with it.

Couldn't delete Packages/com.microsoft.azure.remote-rendering/Plugins/Android/ARM64/libXrApiLayer_MicrosoftHolographicHybridRemoting.so.meta because it's in an immutable folder.
```

**Additional context**

The warnings are spurious and can be ignored. I'm using Unity 2021, but they are likely also present in Unity 2020. I've tested version `1.0.31` and `1.0.54` of this package, both expose the problem.

**Comment by ChristopherManthei** [comment]
Hi @NoTuxNoBux ,
sorry about that. We have fixed the issue and the removal of these placeholders will go out with the next release.
Cheers,
Christopher

**Comment by FlorianBorn71** [comment]
This has been fixed with today's [release 1.0.57](https://github.com/Azure/azure-remote-rendering/releases/tag/1.0.57).
Thanks for reporting!

**Comment by NoTuxNoBux** [comment]
@FlorianBorn71 Is it possible that this is still not fixed? I upgraded today, but they are still there :smile: :

![afbeelding](https://user-images.githubusercontent.com/73220431/165955762-90ea9674-ccbd-42d8-bb09-525218049a63.png)

**Comment by FlorianBorn71** [comment]
Hi @NoTuxNoBux!
Thanks for letting us know, shame on us! ;-)
I have re-opened this ticket and we'll fix it for course. Unfortunately, the next release is already in flight, so it will be in the one after that. But I figure you are not blocked by this, right?
Best Regards!
Florian

**Comment by NoTuxNoBux** [comment]
Hi @FlorianBorn71, thanks for the response and no problem - it happens :smile: .

And indeed, I'm not blocked by this, but just reported it because I noticed it and it is a minor nuisance :slightly_smiling_face: .

**Comment by ChristopherManthei** [comment]
Hi @NoTuxNoBux ,
we just released [ARR 1.0.60](https://github.com/Azure/azure-remote-rendering/releases/tag/1.0.60). We are feeling a bit more confident this time that the issue was actually fixed. Please give it a try!
Cheers,
Christopther

**Comment by NoTuxNoBux** [comment]
@ChristopherManthei Thanks for the response. I did some testing with the same project and it indeed seems to be fixed, thanks!
